### PR TITLE
KOGITO-408: [DMN Designer] Unit tests: kie-wb-common-dmn-webapp-kogito-testing

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/KOGITO-408

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko This PR is ready for review now.

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing/src/test/java/org/kie/workbench/common/dmn/showcase/client/navigator/DMNDiagramsNavigatorScreenTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import org.uberfire.backend.vfs.Path;
+
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DMNDiagramsNavigatorScreenTest {
+
+    private static final String NAME = "name";
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing/src/test/java/org/kie/workbench/common/dmn/showcase/client/navigator/DMNDiagramsNavigatorScreenTest.java]: Name of what is this please? could we update the constant name to be more descriptive?

**Comment by manstis** [comment]
@jomarko Update to `NAME` pushed.

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing/src/test/java/org/kie/workbench/common/dmn/showcase/client/navigator/DMNDiagramsNavigatorScreenTest.java (hunk 1 above)
Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing/src/test/java/org/kie/workbench/common/dmn/showcase/client/navigator/DMNDiagramsNavigatorScreenTest.java]: Updated in new commit.

**Pull request merged**